cmd/commands/universe.go

```diff
@@ -1696,6 +1697,54 @@ func fetchSupplyCommit(ctx *cli.Context) error {
 	return nil
 }
 
+var fetchSupplyLeavesCmd = cli.Command{
+	Name: "fetchleaves",
```

Comment:
nit: doesn't match command name in API docs prefix of the proto file. IMO this should just be `supplyleaves`.